modules/polling/components/DateFilter.tsx

```diff
@@ -45,7 +45,7 @@ export default function DateFilter(props): JSX.Element {
       {...props}
     >
       <Grid gap={2} columns="max-content max-content" sx={{ alignItems: 'baseline' }}>
-        <Text>After</Text>
+        <Text>Poll expired after:</Text>
```

Comment:
Same here, feel like it could be shortened to just `Ended after`, (I think end is more consistent with the wording elsewhere than expired)